js/main.js

```diff
@@ -28,25 +29,31 @@ const windowMgr = require('./windowMgr.js');
 app.on('ready', getUrlAndOpenMainWindow);
 
 function getUrlAndOpenMainWindow() {
-    getConfig().then(function(config) {
-        let protocol = '';
-        // add https protocol if none found.
-        let parsedUrl = nodeURL.parse(config.url);
-        if (!parsedUrl.protocol) {
-            protocol = 'https';
-        }
-        var url = nodeURL.format({
-            protocol: protocol,
-            slahes: true,
-            pathname: parsedUrl.href
+    getRegistry('PodUrl')
+    .then(createWin).catch(function (){
```

Comment:
this promise chain is bit hard to follow...
think it might be better to move getRegistry call inside of getConfig and replace returned url there with one from registry (if window and presents).

also make sure unit tests still run: npm run test